bin/ka-clone

```diff
@@ -482,31 +489,30 @@ def _chomp(s, sep):
 def _cli_process_current_dir(cli_args):
     die_if_not_valid_git_repo()
 
-    backup_existing_hooks()
-
     if not cli_args.no_email:
         set_email(args.email)
-    if not cli_args.no_lint:
-        install_pre_push_lint_hook()
-    if not cli_args.no_msg:
+
+    if cli_args.msg:
         link_commit_template()
-    if not cli_args.no_gitconfig:
+
+    if cli_args.gitconfig:
         link_gitconfig_khan()
 
+    backup_existing_hooks()
+
+    if cli_args.lint:
+        install_pre_push_lint_hook()
+
     if cli_args.protect_master:
         protect_master()
 
     if cli_args.lint_commit:
         _install_git_hook('commit-msg.lint', 'commit-msg.lint')
         _cli_log_step_success("Added commit-msg lint hook")
-    else:
-        _remove_git_hook('commit-msg.lint')
```

Comment:
We don't need to do this anymore because we're moving the whole git hooks folder now to back it up